main/EDParamsMod.F90

```diff
@@ -62,7 +62,7 @@ module EDParamsMod
    real(r8),protected, public :: ED_val_canopy_closure_thresh ! site-level canopy closure point where trees take on forest (narrow) versus savannah (wide) crown allometry
    integer,protected, public  :: stomatal_model  !switch for choosing between stomatal conductance models, 1 for Ball-Berry, 2 for Medlyn
 
-   logical,protected, public :: active_crown_fire        ! flag, 1=active crown fire 0=no active crown fire
+   logical,protected, public :: active_crown_fire_switch  ! flag, 1=active crown fire 0=no active crown fire
```

Comment:
Added suffix `_switch` to prevent compiler error that complained about same name as `subroutine active_crown_fire`.